demo/app/main-view-model-android.ts

```diff
@@ -0,0 +1,42 @@
+import {Observable} from "data/observable";
+var loadingIndicator = require("nativescript-loading-indicator");
+
+export class LoadingAndroidTest extends Observable {
+  
+  public showLoader() {
+    loadingIndicator.show({ message: 'Loading test...' });
```

Comment:
Small thing, but could we use double quotes for strings throughout the project? This is a {N} convention that we're trying to keep to.